src/In.ProjectEKA.HipService/Program.cs

```diff
@@ -47,5 +58,14 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args)
                 .UseSerilog()
                 .UseStartup<Startup>();
         }
+        
+        private static ElasticsearchSinkOptions ConfigureElasticSink(IConfigurationRoot configuration, string environment)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'ConfigureElasticSink'.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5420628)